src/core/lib/security/credentials/tls/grpc_tls_certificate_provider.cc

```diff
@@ -101,7 +101,7 @@ gpr_timespec TimeoutSecondsToDeadline(int64_t seconds) {
 
 FileWatcherCertificateProvider::FileWatcherCertificateProvider(
     std::string private_key_path, std::string identity_certificate_path,
-    std::string root_cert_path, unsigned int refresh_interval_sec)
+    std::string root_cert_path, int64_t refresh_interval_sec)
```

Comment:
uint64_t probably makes more sense here

Reply:
We pass it to `gpr_time_xxx` functions where int64_t is consistently used, so maybe it’s better to use int64_t here as well?